documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -614,3 +614,163 @@ Python collects any number of keyword arguments into a dictionary.  In
+
+pass
+^^^^
+
++--------------------------------------------------+---------------------------------------------------+
+| Python                                           | Dylan                                             |
++==================================================+===================================================+
+| .. code-block:: python                           | .. code-block:: dylan                             |
+|                                                  |                                                   |
+|    if some_condition:                            |    if (some-condition)                            |
+|      pass                                        |      // do nothing                                |
+|                                                  |    end                                            |
++--------------------------------------------------+---------------------------------------------------+
+
+match
+^^^^^
+
++--------------------------------------------------+---------------------------------------------------+
+| Python                                           | Dylan                                             |
++==================================================+===================================================+
+| .. code-block:: python                           | .. code-block:: dylan                             |
+|                                                  |                                                   |
+|    def http_error(status):                       |    define function http-error(status)=>(msg)      |
+|      match status:                               |      select (status)                              |
+|        case 400:                                 |        400 => "Bad request";                      |
+|          return "Bad request"                    |        404 => "Not found";                        |
+|        case 404:                                 |        otherwise => "Something went wrong";       |
+|          return "Not found"                      |      end                                          |
+|        case _:                                   |    end                                            |
+|          return "Something went wrong"           |                                                   |
++--------------------------------------------------+---------------------------------------------------+
+
++--------------------------------------------------+---------------------------------------------------+
+| Python                                           | Dylan                                             |
++==================================================+===================================================+
+| .. code-block:: python                           | Dylan has no pattern matching                     |
```

Comment:
How about "Dylan has no built-in pattern matching but "match" can be implemented in a library as a macro."

I'm not super familiar with Python's `match` so I don't know for sure if a macro could do the exact same thing, but it could certainly do a lot of it.